### DISCUSSION TITLE
Welcome to darkblowV42-Website Discussions!

### DISCUSSION THREAD

**Discussion by imamrasyid** [DarkblowPB/darkblowV42_Website#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **Announce to your community** that Discussions is available! Go ahead and send that tweet, post, or link it from the website to drive traffic here.

  🔗 If you use issue templates, **link any relevant issue templates** such as questions and community conversations to Discussions. Declutter your issues by driving community content to where they belong in Discussions. If you need help, here's a [link to the documentation](https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser).

  ➡️ You can **convert issues to discussions** either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by yuliusardian** [comment]
Halo Om, Since we both from Indonesia,

Are you make your own server ? I've been playing this game since 2009-2010 when I was 7th grade, have you heard/remember about Redox Packet Editor ? If so, I got some benefit from that tool HAHAHA *EvilLaugh*. Recently I got interest on " How's  this game is working " I'm curious since I saw videos from [this channel](https://www.youtube.com/c/LiveOverflowCTF/videos) about reverse engineering game Pwn Adventure 3. So, How can I start ? I've some basic C#.

**Reply by imamrasyid** [reply]
actually i never do any reconstruct the game client, anyway that was a great idea. this game already have public server and ready to use
you can find it on internet, or contact me on discord Lolsecs#0001
